dcurve/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@dgrants/dcurve",
+  "version": "0.0.1",
+  "private": true,
+  "description": "distribution generator for GrantCLR in dgrants",
+  "keywords": [
+    "pairwise",
+    "linear",
+    "CLR",
+    "curve"
+  ],
+  "main": "src/index.ts",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "clean": "rimraf dist",
+    "dev": "echo 'TODO'",
+    "build": "echo 'TODO'",
+    "test": "echo 'TODO'",
```

Comment:
yup ! we'd want to add this in soon

Reply:
@thelostone-mc Is there an issue for this? Just looked but might have missed it

Reply:
@mds1 - I've just opened #137 and added some questions there to start the conversation - it might be better to track this in a discussion until we iron out the details though?